server/package.json

```diff
@@ -25,8 +26,31 @@
     "web-tree-sitter": "^0.20.8"
   },
   "scripts": {
+    "prepack": "cd .. && [ -d client/resources ] || scripts/fetch-docs.sh && cd server && cp -r ../client/resources ../client/tree-sitter-bitbake.wasm . && tsc --outDir out",
```

Comment:
make sure the resources don't get duplicated in the .vsix package which would be dead weight.

Reply:
OK, Fixed.

Reply:
I have moved the duplicated resources to symlinks.